### PR TITLE
kneed 0.8.5 ❄️

### PR DESCRIPTION
**Pull request by psteyer**
kneed 0.8.5 ❄️

**Destination channel:** Snowflake

### Links

- [PKG-4122](https://anaconda.atlassian.net/browse/PKG-4122) 
- [Upstream repository](https://github.com/arvkevi/kneed)

### Explanation of changes:

- bumping version number
- updating SHA
- removing noarch
- adding script option
- adding pinning for numpy
- adding pinning for scripty
- added hatchling dependency
- added `pip` test requires 
- added `pip check` test command
- added package description
- updated home url
- added dev url
- added doc url

[PKG-4122]: https://anaconda.atlassian.net/browse/PKG-4122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
+    - numpy >=1.14.2
+    - scipy >=1.0.0
     - scikit-learn
 test:
   imports:
     - kneed
+  requires:
+    - pip
+  commands:
+  - pip check
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
  commands:
    - pip check
  
```

Comment (lorepirri) [reply recipe/meta.yaml]: space and new line :)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -vv
 
 requirements:
   host:
     - python
     - setuptools
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
```

`setuptools` and `hatchling` are two building systems, in this case, the upstream switched to `hatchling`, so you don't need anymore `setuptools`

https://github.com/arvkevi/kneed/blob/01c2aab60c3348ba2cda44b9385eaee7522f2434/pyproject.toml#L1-L3

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
+  home: https://github.com/arvkevi/kneed
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: Knee point detection using Python
+  description: |
+    This repository is an attempt to implement the kneedle algorithm. 
+    Given a set of x and y values, kneed will return the knee point of the function. 
+    The knee point is the point of maximum curvature.
+  doc_url: https://kneed.readthedocs.io/en/stable/
```
Comment (lorepirri) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://kneed.readthedocs.io
```

**Review by lorepirri — COMMENTED** [review]
Good work on this one, thanks! A few small changes and it's good to go

File: recipe/meta.yaml (hunk 2 above)
Comment (danpetry) [reply recipe/meta.yaml]: do you need wheel as well? usually that's explicit, for reasons I don't know

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,38 +1,48 @@
...
     - setuptools
     - pip
+    - hatchling
   run:
     - python
-    - numpy
-    - scipy
+    - numpy >=1.14.2
+    - scipy >=1.0.0
     - scikit-learn
```
Comment (danpetry) [inline recipe/meta.yaml]: scikit-learn isn't a dep upstream

**Review by danpetry — APPROVED** [review]

**Review by Jrice1317 — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]: only when there's `setuptools` as far as I know

**Review by lorepirri — APPROVED** [review]
LGTM! Thanks for addressing the changes

**Pull request merged**